Resources/Prototypes/_NF/PointsOfInterest/nfsd.yml

```diff
@@ -12,11 +31,11 @@
         - type: StationJobs
           availableJobs:
             Sheriff: [ 1, 1 ]
-            Bailiff: [ 0, 0 ]
-            SeniorOfficer: [ 0, 0 ]
+            Bailiff: [ 1, 1 ]
+            SeniorOfficer: [ 1, 1 ]
             Brigmedic: [ 0, 0 ]
             DetectiveNF: [ 0, 0 ]
-            Deputy: [ 0, 0 ]
+            Deputy: [ 3, 3 ]
```

Comment:
This is an increase over the current amount (2).  Is this intended?

Reply:
yea going to start it with 3